pkg/kubelet/kubelet.go

```diff
@@ -155,6 +155,14 @@ const (
 	// MaxContainerBackOff is the max backoff period for container restarts, exported for the e2e test
 	MaxContainerBackOff = v1beta1.MaxContainerBackOff
 
+	// reducedMaxContainerBackOff is the default max backoff period for container restarts when the alpha feature
+	// gate ReduceDefaultCrashLoopBackOffDecay is enabled
+	reducedMaxContainerBackOff = 30 * time.Second
+
+	// reducedBaseContainerBackOff is the default initial backoff period for container restarts when the alpha feature
+	// gate ReduceDefaultCrashLoopBackOffDecay is enabled
+	reducedBaseContainerBackOff = 1 * time.Second
```

Comment:
nit: I prefer "initial" to "base"

Reply:
And maybe rename `containerBackOffPeriod` so that the naming is consistent, and move it to here (or next to MaxContainerBackoff)

Reply:
Actually, can you also add `CrashLoop` to the name of all these constants?